packages/cli/src/index.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 // 3p
+import chalk from 'chalk';
```

Comment:
Same as in https://github.com/FoalTS/foal/pull/307#discussion_r247313481

Reply:
👍